modules/classes.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
```

Comment:
- [ ] As requeted by the previous reviewer, kindly make sure to no disable linter for a whole file. You can disable or customize one particular rule in a file.

  At this moment you are facing this error of `File has too many classes (2). The maximum allowed is 1           max-classes-per-file` which you can customize by making sure the file can be able to hold two classes at the same time.

   Kindly use this line instead `/* eslint max-classes-per-file: ["error", 2] */`. The error will go away eventually.